bazel/api-golden/index_npm_packages.ts

```diff
@@ -49,7 +49,7 @@ async function main(
     // entry-point we maintain a separate golden file. These golden files are
     // based on the name of the defining NodeJS exports subpath in the NPM package,
     // See: https://api-extractor.com/pages/overview/demo_api_report/.
-    const goldenName = join(subpath, 'index.md');
+    const goldenName = join(subpath, 'index.api.md');
```

Comment:
For reference, this came up because all the golden files in the FW repo got renamed to `.api.md` after the latest update. It seems like we were supposed to do this all along, based on https://api-extractor.com/pages/configs/api-extractor_json/#apireportreportfilename